presentation/diarywrite/src/main/java/com/hilingual/presentation/diarywrite/component/RecommendedTopicDropdown.kt

```diff
@@ -0,0 +1,113 @@
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.hilingual.core.common.extension.noRippleClickable
+import com.hilingual.core.designsystem.theme.HilingualTheme
+import com.hilingual.presentation.diarywrite.R
+
+@Composable
+internal fun RecommendedTopicDropdown(
+    engTopic: String,
+    korTopic: String,
+    modifier: Modifier = Modifier
+) {
+    var isKor by remember { mutableStateOf(false) }
+    var expanded by remember { mutableStateOf(false) }  // 확장 여부
+
+    Column(
+        modifier = modifier
+            .clip(RoundedCornerShape(8.dp))
+            .background(HilingualTheme.colors.gray100)
+            .fillMaxWidth()
+            .padding(12.dp),
+        verticalArrangement = Arrangement.spacedBy(16.dp)
+    ) {
+        // '오늘의 추천 주제 참고하기' 영역
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.SpaceBetween
+        ) {
+            Text(
+                text = "오늘의 추천 주제 참고하기",
+                style = HilingualTheme.typography.bodyM14,
+                color = HilingualTheme.colors.gray700
+            )
+            Icon(
+                modifier = Modifier
+                    .size(24.dp)
+                    .graphicsLayer {
+                        scaleY = if (expanded) -1f else 1f
+                    }
+                    .noRippleClickable { expanded = !expanded },
+                imageVector = ImageVector.vectorResource(R.drawable.ic_arrow_down_20),
+                contentDescription = null,
+                tint = HilingualTheme.colors.gray400
+            )
+        }
+
+        // 드롭다운 활성화되었을 때 보이는 추천 주제 영역
+        if (expanded) {
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .clip(RoundedCornerShape(8.dp))
+                    .background(HilingualTheme.colors.white)
+                    .padding(12.dp),
+                verticalAlignment = Alignment.CenterVertically,
+                horizontalArrangement = Arrangement.SpaceBetween
+            ) {
+                Text(
+                    text = if (isKor) {
+                        korTopic
+                    } else {
+                        engTopic
+                    },
```

Comment:
```suggestion
                Text(text = if (isKor) korTopic else engTopic)
```
p3
요렇게 사용하는게 코틀린을 더 활용하는 방법입니두ㅎㅎ